{"name": "test_valid_create_new_note", "status": "broken", "statusDetails": {"message": "KeyError: 'id'", "trace": "@allure.id(\"Vadim kandeev. Test_3\")\n    @allure.story(\"Создание записи нового сотрудника\")\n    @allure.epic(\"Сотрудники\")\n    def test_valid_create_new_note():\n    \n        company_id = 277\n        request_text = \"select first_name, last_name, middle_name, phone, avatar_url, \\\"isActive\\\", id from employee e where id = :curent_id \"\n        api_resp_first_name = \"Dart\"\n        api_resp_last_name = \"Waider\"\n        api_resp_middle_name = \"Sith\"\n        api_resp_phone = \"87776665544\"\n        api_resp_url = \"www.StarWars.com\"\n        api_resp_new_worker = driver.employee_post(company_id, api_resp_first_name, api_resp_last_name, api_resp_middle_name, api_resp_phone, api_resp_url)\n>       api_resp_new_worker_id = api_resp_new_worker[0]['id'] # записываем новый id в переменную, чтобы создать запрос в БД\nE       KeyError: 'id'\n\ntest_employee_page_1.py:58: KeyError"}, "start": 1677862788861, "stop": 1677862789485, "uuid": "8d08ac7d-8fb7-41f9-a16d-8d33eb983f43", "historyId": "c655cf5adf7eb7d6627cb940beabbf14", "testCaseId": "ab43fa5ed4bf622ceeaade7e9a6ab374", "fullName": "test_employee_page_1#test_valid_create_new_note", "labels": [{"name": "epic", "value": "Сотрудники"}, {"name": "as_id", "value": "Vadim kandeev. Test_3"}, {"name": "story", "value": "Создание записи нового сотрудника"}, {"name": "suite", "value": "test_employee_page_1"}, {"name": "host", "value": "VAdims-Mac-mini.local"}, {"name": "thread", "value": "4384-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_employee_page_1"}]}